careers/technical-engineer.md

```diff
@@ -1,73 +1,99 @@
+- Advocating for effective solutions and strategies
+- Bringing perspective from other teams or departments
 
-**Responsibilities:**
+**Mentorship:**
+
+A Staff Engineer is far more likely to change your company's long-term trajectory by growing other engineers than through personal heroics. Their effort here leads to a well-trained team with multiple experts in each domain.
+
+Effective mentorship is not a checkbox on the career ladder that can be satisfied by simply sharing advice and experience. Staff engineers build an ongoing relationship, understand the context of their mentee(s), and [sponsor](https://larahogan.me/blog/what-sponsorship-looks-like/) them.
+
+Concrete examples could include:
+
+- Owning onboarding and continuing education initiatives
+- No bus-factor-one concerns / building redundancy in systems and knowledge
+- Providing critical feedback on architectural design documents
+- Encouraging other engineers to take a specific online training or certification course
+
+**Exploration:**
 
-* Track metrics to ensure code quality standards
-* Advocate for improvements to product quality, security, and performance
-* Maintain a comprehensive technical knowledge of our entire software system
-* Assume responsibility for fixing problems / "Always Responsible Individual"
-* Participate in Tier 2 on-call rotations
-* Managing projects across multiple trimesters
+Our teams are already optimized to incrementally improve. The role of the staff engineer is to be *transformative*. A staff engineer does more than improve their part of the codebase; they solve ambiguous, important problems that the company's current systems are ill-shaped to address. They tackle impossible technical problems (high ambiguity, no in-house expertise, poorly defined). They identify and reduce existential risks. They look beyond their team and project to address organization-level problems (2-3 year growth, org health). They set the technical direction for a department.
+
+Concrete examples could include:
+
+- Reduce infrastructure costs by an order of magnitude
+- Identify a multi-region strategy that takes six months instead of three years
+- Addressing the sudden realization that your primary database only has three months of remaining disk space, and you can't upgrade to a larger size
+- Altering our CI/CD strategy to improve DORA metrics by 50% across the board
+- Identifying a software vendor that reduces churn and increases revenue by 10%
+
+**Be Glue:**
+
+Perhaps most difficult to define, Staff Engineers do [glue work](https://noidea.dog/glue). They see the important things that need done and make sure they happen (whether doing it themselves or finding the right person or process for the job).
+
+Concrete examples could include:
+
+- decreasing onboarding time
+- getting the team to agree on coding standards
+- reviewing designs and architecture decisions
+- making us build the right thing
+- making our customers happier
```

Comment:
These are not concrete, so I would make them so or drop them from the list of concrete examples.